ssh-krb-node-executor-plugin/plugin.yaml

```diff
@@ -22,13 +23,22 @@ providers:
           title: Kerberos keytab
           type: String
           required: true
-          description: "The path and filename to the Kerberos key table used to authenticate the user (e.g. /tmp/user.ktab)"
+          description: "The path and filename to the Kerberos keytab used to authenticate the user (e.g. /tmp/user.ktab)"
+        - name: use_kerberos_custom_cache_file (recommanded)
```

Comment:
If I read correctly the [documentation](https://docs.rundeck.com/docs/developer/01-plugin-development.html#script-plugin-development), the `name` key in a config entry contains a value that's machine readable which might eventualy map to a Rundeck variable name that can be consumed later by the plugin for instance as an environment variable with the form `$RD_CONFIG_FOO`. Doesn't adding that `(recommanded)` substring there break things?

Perhaps the note about it being recommended or not (to be discussed) should go into the `description` which seems to be solely meant to be consumed by humans?

Reply:
Yes please, add that to the description.